sw/ground_segment/tmtc/server_globals.ml

```diff
@@ -8,6 +8,7 @@ let port = ref 8889
 (** FIXME: Should be read from messages.xml *)
```

Comment:
I think that the ap modes should be read from messages.xml, but It isn't implemented right now.